pkg/cli/options.go

```diff
@@ -33,3 +34,67 @@ func ValidateOptions(opts apis.Validatable) func(cmd *cobra.Command, args []stri
 		return nil
 	}
 }
+
+type ListOptions struct {
+	Namespace     string
+	AllNamespaces bool
+}
+
+func (opts *ListOptions) Validate(ctx context.Context) *FieldError {
+	errs := &FieldError{}
+
+	if opts.Namespace == "" && !opts.AllNamespaces {
```

Comment:
my initial reaction when reading this: shouldn't we allow an empty namespace for the default namespace case?
after examining the codebase further, it may be worth documenting that this function is typically called in commands' PreRunE block and the default namespace is usually already injected prior to that PreRunE block (typically via the `NamespaceFlag` function)

Reply:
This is options validation, not flag or args validation. The flags and args have already been bound to the options before this code runs. I'll add a note about the options lifecycle.

Reply:
done

Reply:
👍